stable/opa/values.yaml

```diff
@@ -45,7 +45,23 @@ admissionControllerKey: ""
 
 # Docker image and tag to deploy.
 image: openpolicyagent/opa
-imageTag: 0.10.1
+imageTag: 0.10.2
+
+mgmt:
+  enabled: true
+  image: openpolicyagent/kube-mgmt
+  imageTag: 0.8-dev
```

Comment:
We should cut a release so that we've got a stable tag to refer to here. Do you have any changes you want to go into 0.8 before we do that?

Reply:
I don't know of anything. With maybe the exception of, if it doesn't support contacting opa via unix socket, if that can be added quick before 0.8 that would be cool.